setup.ts

```diff
@@ -108,10 +120,7 @@ async function redisConfiguration(): Promise<void> {
 
     const config = dotenv.parse(fs.readFileSync(".env"));
     config.REDIS_URL = url;
```

Comment:
1. I commented out the Redis URL and then ran the script
1. When prompted I used localhost for the hostname
1. I then used port 1235 as the port which was incorrect
1. The script said that Redis was functional
1. The script completed with the congratulations message
1. The script did not update the Redis URL

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/c2613bdb-95a1-4f4d-961d-56655baf2033)

Reply:
The Redis check is not functional. It detects Redis running on google.com

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/9c072a2f-24f6-44d1-99d5-3ce30dc58c60)